src/components/ErrorHandling/Sentry.tsx

```diff
@@ -39,10 +40,16 @@ const initializeSentry = () => {
 
 const isInitialized = () => !!getCurrentHub().getClient();
 
+type ErrorMetadata = {
+  gqlErr?: ErrorResponse["graphQLErrors"][0];
+  operationName?: ErrorResponse["operation"]["operationName"];
+  variables?: ErrorResponse["operation"]["variables"];
+};
+
 const sendError = (
   err: Error,
   severity: SeverityLevel,
-  metadata?: { [key: string]: any },
+  metadata?: ErrorMetadata,
```

Comment:
This typing is technically broken since https://github.com/evergreen-ci/spruce/blob/45550f1a6d19b1825ec5beeed7f8ada29c857f99/src/utils/errorReporting.ts#L10 passes a generic `{ [key:string]: any }` so this type is never actually used.

Reply:
It is also not flexible enough to allow for any metadata to be passed in once the typing in reportError is fixed.

Reply:
Updated the typing in `errorReporting.ts`. If you want strict typing I think we should just go all the way and update this type as more metadata fields are added rather than allow the type to be flexible.